charts/newrelic-infrastructure/values.yaml

```diff
@@ -133,6 +136,9 @@ ksm:
     - operator: "Exists"
       effect: "NoExecute"
   nodeSelector: {}
+  # -- (bool) Sets pod's hostNetwork. Uncomment to configure hostnetwork to ksm deployment only, bypasses global/common variable 
+  # @default ==  `global.hostNetwork`
+  #hostNetwork: false
```

Comment:
Same as above, I think helm-docs does not understand this syntax.

Reply:
done